content/blueprints/spec-upload-resources.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: bt_wiki
+uid: workflows section
+title: Upload Resources
+category: Blueprints
+draft: false
+weight: 1900
+
+---
+
+# Declaration
+The upload_resources section in the DSL is a dictionary comprises two subsections:
+
+plugin_resources: designated for uploading plugins to the manager.
+dsl_resources: designated for uploading resources needed for the parsing process of blueprints.
+ 
+ 
+upload_resources:
+    plugin_resources: 
+     - ...
+    dsl_resources: 
+     - 'source_path': ...
+       'destination_path': ...
+ 
+ 
+# Definition
+Keyname	            |       Required	|   Type	    | Description
+--------------------|-------------------|---------------|-------------------------
+plugin_resources    | no	            |   list	    | A list of plugins (URLs or local paths) to be uploaded to the manager.
+dsl_resources	    | no	            |   dict	    | A list of dictionaries each comprises a source_path and destination_path for each dsl_resource.
+source_path	        | yes	            |   string	    | The source path for the dsl resource.
+destination_path    | yes	            |   string	    | A relative destination path for the resource (relative to the file server home dir).
+
+{{% gsNote title="Note" %}}
+All plugins uploaded to the manager should be in wgn format.
+{{% /gsNote %}}
+
+
+# Examples
+ 
+upload_resources:
+    plugin_resources: 
+     - 'http://www.my-plugin.com/path/to/plugin.wgn'
+    dsl_resources: 
+     - 'source_path': 'http://www.my-plugin.com/path/to/plugin.yaml'
+       'destination_path': '/path/to/local/plugin.yaml'
+ 
+ 
```

Comment:
I think this section requires some more explanation. It isn't clear which resources could be uploaded and why.